crates/containerd-shim-wasm/src/sandbox/instance.rs

```diff
@@ -97,15 +98,15 @@ impl InstanceConfig {
         self.containerd_address.clone()
     }
 
-    /// set the systemd cgroup for the instance
-    pub fn set_systemd_cgroup(&mut self, systemd_cgroup: bool) -> &mut Self {
-        self.systemd_cgroup = systemd_cgroup;
+    /// set the options config for the instance
+    pub fn set_options_config(&mut self, options_config: OptionsConfig) -> &mut Self {
```

Comment:
Ah, I always disliked that this struct has accessor methods instead or public members. But since that's the pattern here, lets just go with it 🥲

Reply:
Being unfamiliar with Rust or runwasi, I don't know why you want to remove accessor methods, but I could do this in a separate PR.

Reply:
Don't worry with the accessors in `InstanceConfig`, that would need a wider refactor.
Thanks for contributing :-)